docs/features/W3cCredentials.md

```diff
@@ -0,0 +1,633 @@
+        {
+            "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
+        }
+    ]
+}
+```
+
+#### Signature Suite
+
+Select a signature suite for use. VC-DI format currently supports EdDSA signature suites for issuing credentials.
+
+- [`Ed25519Signature2020`](https://w3c.github.io/vc-di-eddsa/#ed25519signature2020-0)
+
+#### DID Method
+
+Choose a DID method for issuing the credential. VC-DI format currently supports the `did:key` method.
+
+##### `did:key`
+
+A `did:key` did is not anchored to a ledger, but embeds the key directly in the identifier part of the did. See the [did:key Method Specification](https://w3c-ccg.github.io/did-method-key/) for more information.
+
+You can create a `did:key` using the `/wallet/did/create` endpoint with the following body.
+
+```jsonc
+{
+  "method": "key",
+  "options": {
+    "key_type": "ed25519"
+  }
+}
+```
+
+### Issue a Credential
+
+The issuance of W3C credentials is facilitated through the `/issue-credential-2.0/send` endpoint. This process adheres to the formats described in [RFC 0809 VC-DI](https://github.com/hyperledger/aries-rfcs/blob/main/features/0809-w3c-data-integrity-credential-attachment/README.md) and utilizes `didcomm` for communication between agents.
+
+To issue a W3C credential, follow these steps:
+
+1. **Prepare the Credential Data:**
+Ensure the credential data conforms to the VC-DI context.
+
+<details>
+<summary>JSON example</summary>
+
+```jsonc
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/security/data-integrity/v2",
+    {
+      "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
+    }
+  ],
+  "type": ["VerifiableCredential"],
+  "issuer": "did:key:z6MkqG......",
+  "issuanceDate": "2023-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:key:z6Mkh......",
+    "name": "John Doe"
+  },
+  "proof": {
+    "type": "Ed25519Signature2020",
+    "created": "2023-01-01T00:00:00Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:key:z6MkqG......#z6MkqG......",
+    "jws": "eyJhbGciOiJFZERTQSJ9..."
+  }
+}
+```
+</details>
+
+2. **Select Credential type**
+The ability to choose the credential type (indy, vc_di) to be issued. The credential type is used to determine the schema for the credential data.
+
+The format to change credential can be seen in the ["Demo Instruction"](https://github.com/sarthakvijayvergiya/aries-cloudagent-python/blob/whatscookin/feat/vc-di-proof/docs/demo/README.md)
+
+3. **Send the Credential:**
+Use the `/issue-credential-2.0/send` endpoint to issue the credential.
+
+<details>
+<summary>JSON example</summary>
+
+```jsonc
+{
+  "auto_issue": true,
+  "auto_remove": false,
+  "comment": "Issuing a test credential",
+  "credential_preview": {
+    "@type": "https://didcomm.org/issue-credential/2.0/credential-preview",
+    "attributes": [
+      {"name": "name", "value": "John Doe"}
+    ]
+  },
+  "trace": false
+}
```

Comment:
don't we need to include a filter for issue-credential 2.0?

Reply:
Great one picking this out. I have included filter